docs/dev/perl5/news/2013/perl-5.16.3.html

```diff
@@ -0,0 +1,70 @@
+[% page.title = "Perl 5.16.3 Release Announcement" %]
+<meta charset="utf-8">
```

Comment:
Is this actually necessary?

Our pages should already have 
    `<meta http-equiv="Content-Type" content="text/html;charset=utf-8" />`

in them.